invoices/invoiceregistry.go

```diff
@@ -196,6 +259,29 @@ func (i *InvoiceRegistry) invoiceEventNotifier() {
 				i.singleNotificationClients[e.id] = e
```

Comment:
Can this (htlc not found in the invoice) ever happen though?

Reply:
This comment seems to be misplaced by github. I think this is about `cancelSingleInvoice`? Yes, if we scheduled a timer, the htlc should have been record on the invoice.

Reply:
Hmm, looks like the PR was changed during my review. I'll place this comment to where it belongs.